www/src/py_builtin_functions.js

```diff
@@ -1093,12 +1093,11 @@ function isinstance(obj,arg){
    // Search __instancecheck__ on arg
     var ce = __BRYTHON__.current_exception
```

Comment:
If the try-catch block is removed, the `ce` variable is not used anywhere and could be deleted...